pages/add/form-fields/detail-info-fields.js

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  creators: {
+    type: `text`,
+    label: `Who are the creators? This could be staff, contributors, partners…`,
+    placeholder: `Name`,
+    fieldClassname: `form-control`,
+    multiplicity: 1,
+    addLabel: `+ Add another`,
+    // optional: true
+  },
+  interest: {
+    type: `text`,
+    label: `Why might this be interesting to other people in our network?`,
+    placeholder: ``,
+    fieldClassname: `form-control`,
+    validator: {
+      error: `Maximum 300 characters.`,
+      validate: function(value) {
+        if (!value) {
+          // this is an optional field, it can be left blank
+          return;
+        }
+
+        return value && value.length > 300; // return true if value fails validation
```

Comment:
probably better to do this explicit:
```
let passes = value && value.length > 300;
if (!passes) return new Error("blah!");
```
And we don't need a final return, as the test is "does this function return a truthy value", and a natural function end is equivalent to `return undefined`, so is falsey